src/python/pants/goal/BUILD

```
@@ -2,3 +2,6 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 python_library()
+python_tests(
+  name='tests'
+)
```

Super nitty, feel free to ignore. 
```suggestion
python_tests(name='tests')
```